SwiftEvolve/Tests/SwiftEvolveTests/RegressionTests.swift

```diff
@@ -25,40 +24,40 @@ class RegressionTests: XCTestCase {
         var p9: Int
       }
       """
-    ) { code in
-      let evo = ShuffleMembersEvolution(mapping: [])
+    )
+    let evo = ShuffleMembersEvolution(mapping: [])
 
-      for node in code.filter(whereIs: MemberDeclListSyntax.self) {
-        let evolved = evo.evolve(node)
-        let evolvedCode = evolved.description
+    for node in code.filter(whereIs: MemberDeclListSyntax.self) {
+      let evolved = evo.evolve(node)
+      let evolvedCode = evolved.description
 
-        let locs = (0...9).compactMap {
-          evolvedCode.range(of: "p\($0)")?.lowerBound
-        }
+      let locs = (0...9).compactMap {
+        evolvedCode.range(of: "p\($0)")?.lowerBound
+      }
 
-        XCTAssertEqual(locs.count, 10, "All ten properties were preserved")
+      XCTAssertEqual(locs.count, 10, "All ten properties were preserved")
 
-        for (prev, next) in zip(locs, locs.dropFirst()) {
-          XCTAssertLessThan(prev, next, "Adjacent properties are in order")
-        }
+      for (prev, next) in zip(locs, locs.dropFirst()) {
+        XCTAssertLessThan(prev, next, "Adjacent properties are in order")
       }
     }
   }
 
   func testStoredIfConfigBlocksMemberwiseInitSynthesis() throws {
-    // FIXME: Crashes when run in Xcode because of a version mismatch between
-    // SwiftSyntax and the compiler it uses (specifically, how they represent
-    // accessor blocks). Should pass in "env PATH=... swift build".
-    try SwiftLang.withParsedCode(
-      """
-      struct A {
-        #if os(iOS)
-          var a1: Int
-        #endif
-        var a2: Int { fatalError() }
-      }
-      """
-    ) { code in
+    do {
+      // FIXME: Crashes when run in Xcode because of a version mismatch between
+      // SwiftSyntax and the compiler it uses (specifically, how they represent
+      // accessor blocks). Should pass in "env PATH=... swift build".
```

Comment:
I actually fixed this in a previous PR (unless it regressed!), but I can remove this comment myself.